src/global/mixins.scss

```diff
@@ -44,3 +44,8 @@
   top: 50%;
   transform: translateY(-50%);
 }
+
+@mixin zeroWidthSpace() {
```

Comment:
Could you maybe add a little documentation as to why this is needed?
And should we maybe inlcude the ::before inside this mixin?